<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Server_end</class>
 <widget class="QDialog" name="Server_end">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>700</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>1200</width>
    <height>700</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>ServerEnd</string>
  </property>
  <property name="windowOpacity">
   <double>0.980000000000000</double>
  </property>
  <widget class="QGroupBox" name="LicenseBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>480</y>
     <width>481</width>
     <height>121</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="title">
    <string>License plate Image</string>
   </property>
   <widget class="QLabel" name="LicenseImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>460</width>
      <height>90</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>460</width>
      <height>90</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>460</width>
      <height>90</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ImageBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>20</y>
     <width>481</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="title">
    <string>Image</string>
   </property>
   <widget class="QLabel" name="VehicleImage">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>18</y>
      <width>461</width>
      <height>383</height>
     </rect>
    </property>
    <property name="minimumSize">
     <size>
      <width>461</width>
      <height>383</height>
     </size>
    </property>
    <property name="maximumSize">
     <size>
      <width>461</width>
      <height>383</height>
     </size>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QTabWidget" name="RecogniseList">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>321</width>
     <height>311</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="currentIndex">
    <number>2</number>
   </property>
   <widget class="QWidget" name="WaitingTab">
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <attribute name="title">
     <string>Waiting</string>
    </attribute>
    <widget class="QTableView" name="WaitingTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="SuccessTab">
    <attribute name="title">
     <string>Success</string>
    </attribute>
    <widget class="QTableView" name="SuccessTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="FailTab">
    <attribute name="title">
     <string>Fail</string>
    </attribute>
    <widget class="QTableView" name="FailTable">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>321</width>
       <height>291</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::HLine</enum>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QGroupBox" name="Moreinfo">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>850</x>
     <y>30</y>
     <width>331</width>
     <height>651</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QGroupBox" name="ViolatingInfoBox2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>311</width>
      <height>351</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
    </property>
    <property name="title">
     <string>Violating vehicle list</string>
    </property>
    <widget class="QTableView" name="ViolatingList">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>311</width>
       <height>331</height>
      </rect>
     </property>
     <property name="minimumSize">
      <size>
       <width>311</width>
       <height>331</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>311</width>
       <height>16777215</height>
      </size>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="QueryBox">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>400</y>
      <width>311</width>
      <height>241</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
    </property>
    <property name="title">
     <string>Query method</string>
    </property>
    <widget class="QCheckBox" name="NameMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>141</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Query by name</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="IDMethod">
     <property name="geometry">
      <rect>
       <x>130</x>
       <y>30</y>
       <width>171</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Query by ID card No.</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="LicenseMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>160</y>
       <width>191</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>Input license plate No.</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QCheckBox" name="TimeMethod">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>271</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>From</string>
     </property>
     <property name="autoExclusive">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLineEdit" name="LicenseNumLine">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>160</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
    </widget>
    <widget class="QLabel" name="TimeMethod_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>120</y>
       <width>21</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>To</string>
     </property>
    </widget>
    <widget class="QPushButton" name="SubmitButton">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>200</y>
       <width>101</width>
       <height>21</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="text">
      <string>Submit</string>
     </property>
     <property name="autoDefault">
      <bool>true</bool>
     </property>
     <property name="default">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>80</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2012</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd HH:mm:ss</string>
     </property>
    </widget>
    <widget class="QDateTimeEdit" name="dateTime_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>120</y>
       <width>194</width>
       <height>22</height>
      </rect>
     </property>
     <property name="dateTime">
      <datetime>
       <hour>0</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2012</year>
       <month>1</month>
       <day>1</day>
      </datetime>
     </property>
     <property name="displayFormat">
      <string>yyyy/MM/dd HH:mm:ss</string>
     </property>
    </widget>
   </widget>
   <widget class="QWidget" name="layoutWidget_3">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>360</y>
      <width>311</width>
      <height>27</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_5">
     <item>
      <widget class="QPushButton" name="DeleteButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Delete</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="EmptyButton">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Empty</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QGroupBox" name="ViolatingInfoBox">
   <property name="geometry">
    <rect>
     <x>360</x>
     <y>600</y>
     <width>481</width>
     <height>81</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="title">
    <string>Violating vehicle information</string>
   </property>
   <widget class="QWidget" name="layoutWidget_4">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>461</width>
      <height>50</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLabel" name="LicenseLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>License plate No.</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="LicenseLine">
         <property name="text">
          <string/>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QLabel" name="TimeLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Violating time</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="TimeLine">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="ViolatingIDLabel">
         <property name="styleSheet">
          <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
         </property>
         <property name="text">
          <string>Violating ID</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="ViolatingIDLine">
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QLabel" name="bgLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: qlineargradient(spread:pad, x1:0.761364, y1:0.08, x2:1, y2:1, stop:0 rgba(0, 78, 152, 255), stop:1 rgba(136, 255, 255, 255));
border-left-style:solid;border-left-width:1px;border-left-color:black;
border-top-style:solid;border-top-width:1px;border-top-color:black;
border-right-style:solid;border-right-width:1px;border-right-color:black;</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="bgLabel_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>201</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">color: rgb(255, 255, 255);
font: 87 8pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>Traffic monitoring system 2012</string>
   </property>
  </widget>
  <widget class="QPushButton" name="ExitButton">
   <property name="geometry">
    <rect>
     <x>1150</x>
     <y>0</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(229, 86, 131);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>X</string>
   </property>
  </widget>
  <widget class="QLabel" name="logoLabel">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>40</y>
     <width>130</width>
     <height>160</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>130</width>
     <height>160</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>130</width>
     <height>160</height>
    </size>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="PortLabel">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>50</y>
     <width>54</width>
     <height>30</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 87 9pt &quot;Arial Black&quot;;
color: rgb(102, 102, 102);</string>
   </property>
   <property name="text">
    <string>Port No.</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="PortLine">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>80</y>
     <width>173</width>
     <height>31</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>173</width>
     <height>20</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 9pt &quot;Calibri&quot;;</string>
   </property>
   <property name="inputMethodHints">
    <set>Qt::ImhNone</set>
   </property>
  </widget>
  <widget class="QPushButton" name="StartButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);
</string>
   </property>
   <property name="text">
    <string>START</string>
   </property>
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="StopButton">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>130</y>
     <width>81</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>STOP</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>30</y>
     <width>341</width>
     <height>651</height>
    </rect>
   </property>
   <property name="title">
    <string/>
   </property>
   <widget class="QTextEdit" name="RunningIfo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>170</y>
      <width>321</width>
      <height>141</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
  </widget>
  <widget class="QPushButton" name="minimizeButton">
   <property name="geometry">
    <rect>
     <x>1130</x>
     <y>0</y>
     <width>21</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(97, 168, 255);
color: rgb(255, 255, 255);
font: 87 9pt &quot;Arial Black&quot;;</string>
   </property>
   <property name="text">
    <string>—</string>
   </property>
  </widget>
  <widget class="QPushButton" name="QueryButton">
   <property name="geometry">
    <rect>
     <x>650</x>
     <y>440</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Query&gt;&gt;</string>
   </property>
  </widget>
  <widget class="QPushButton" name="RecogniseButton">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>420</x>
     <y>442</y>
     <width>131</width>
     <height>23</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 127, 190);
font: 9pt &quot;Calibri&quot;;
color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string>Recognise</string>
   </property>
   <property name="default">
    <bool>false</bool>
   </property>
  </widget>
  <zorder>QueryButton</zorder>
  <zorder>RecogniseButton</zorder>
  <zorder>groupBox</zorder>
  <zorder>LicenseBox</zorder>
  <zorder>ImageBox</zorder>
  <zorder>RecogniseList</zorder>
  <zorder>Moreinfo</zorder>
  <zorder>ViolatingInfoBox</zorder>
  <zorder>bgLabel</zorder>
  <zorder>bgLabel_2</zorder>
  <zorder>ExitButton</zorder>
  <zorder>logoLabel</zorder>
  <zorder>PortLabel</zorder>
  <zorder>PortLine</zorder>
  <zorder>StartButton</zorder>
  <zorder>StopButton</zorder>
  <zorder>minimizeButton</zorder>
 </widget>
 <tabstops>
  <tabstop>PortLine</tabstop>
  <tabstop>StartButton</tabstop>
  <tabstop>StopButton</tabstop>
  <tabstop>QueryButton</tabstop>
  <tabstop>DeleteButton</tabstop>
  <tabstop>EmptyButton</tabstop>
  <tabstop>NameMethod</tabstop>
  <tabstop>IDMethod</tabstop>
  <tabstop>TimeMethod</tabstop>
  <tabstop>LicenseMethod</tabstop>
  <tabstop>LicenseNumLine</tabstop>
  <tabstop>SubmitButton</tabstop>
  <tabstop>LicenseLine</tabstop>
  <tabstop>TimeLine</tabstop>
  <tabstop>ViolatingIDLine</tabstop>
  <tabstop>dateTime</tabstop>
  <tabstop>dateTime_2</tabstop>
  <tabstop>ExitButton</tabstop>
  <tabstop>RecogniseList</tabstop>
  <tabstop>WaitingTable</tabstop>
  <tabstop>ViolatingList</tabstop>
  <tabstop>RunningIfo</tabstop>
  <tabstop>minimizeButton</tabstop>
  <tabstop>RecogniseButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
